scripts/memory-scorecard.sh

```
@@ -0,0 +1,335 @@
+unset RUSTY_BRAIN_DB RUSTY_BRAIN_SOCKET RUSTY_BRAIN_NAMESPACE RUSTY_BRAIN_IDLE_TIMEOUT_SECS
+WORKROOT="$(mktemp -d "${TMPDIR:-/tmp}/rb-scorecard.XXXXXX")"
+RESULTS="${OUT:-$WORKROOT/scorecard.tsv}"; : > "$RESULTS"
+cleanup() { rm -rf "$WORKROOT" 2>/dev/null || true; }
+trap cleanup EXIT
+
+seed_home() { mkdir -p "$1"; printf '{"hasCompletedOnboarding": true}\n' > "$1/.claude.json"; }
+
+run_session() { # home proj prompt log [extra args...]
+  local home="$1" proj="$2" prompt="$3" log="$4"; shift 4
+  (
+    export HOME="$home"
```

Shellcheck (0.11.0)

[info] Modification of HOME is local (to subshell caused by (..) group).

(SC2030)